robot_manager:
  ros__parameters:
    control_mode: 7
    client_ip: "0.0.0.0"
    controller_ip: "0.0.0.0"
    cartesian_velocity_controller_name: "twist_controller"
    cartesian_pose_broadcaster_name: "cartesian_pose_broadcaster"
# Control mode enums:
#  1 - joint position control
#  2 - joint impedance control
#  3 - joint velocity control (not supported yet)
#  4 - joint torque control
#  7 - cartesian velocity (twist) control (only for mobile robots)
# ... others are not yet supported
